ticdc/ticdc-simple-protocol.md

```diff
@@ -0,0 +1,716 @@
+    - If a table does not receive any new DML messages within 30 minutes, the table is considered inactive. TiCDC stops sending `BOOTSTRAP` events for the table until new DML events are received.
+- Destination: By default, TiCDC sends `BOOTSTRAP` events to all partitions of the corresponding topic. You can adjust the sending strategy by configuring the `send-bootstrap-to-all-partition` parameter in the sink configuration.
+
+## Message consumption methods
+
+Because the TiCDC Simple protocol does not include the schema information of the table when sending DML messages, the downstream needs to receive DDL or BOOTSTRAP messages and cache the schema information of the table when consuming DML messages. When receiving a DML message, the downstream can obtain the corresponding table schema information through the `table` name and `schemaVersion` field in the DML message, and then correctly consume the DML message.
+
+The following describes how to correctly consume DML messages based on DDL or BOOTSTRAP messages. According to preceding descriptions, the following information is known:
+
+- Each DML message contains a `schemaVersion` field to mark the schema version number of the table corresponding to the DML message.
+- Each DDL message contains a `tableSchema` and `preTableSchema` field to mark the schema information of the table before and after the DDL event.
+- Each BOOTSTRAP message contains a `tableSchema` field to mark the schema information of the table corresponding to the BOOTSTRAP message.
+
+The consumption methods are introduced in the following two scenarios.
+
+### Scenario 1: The consumer starts consuming from the beginning
+
+In this scenario, the consumer starts consuming from the beginning, so the consumer can receive all DDL and BOOTSTRAP messages of the table. In this case, the consumer can obtain the schema information of the table through the `table` name and `schemaVersion` field in the DML message. The detailed process is as follows:
+
+![TiCDC Simple Protocol consumer scene 1](/media/ticdc/ticdc-simple-consumer-1.png)
+
+### Scenario 2: The consumer starts consuming from the middle
+
+When a new consumer joins the consumer group, it might start consuming from the middle, so it might miss the previous DDL and BOOTSTRAP messages. In this case, the consumer might receive some DML messages before obtaining the schema information of the table. Therefore, the consumer needs to wait for a period of time until it receives the DDL or BOOTSTRAP message to obtain the schema information of the table. Because TiCDC sends BOOTSTRAP messages periodically, the consumer can always obtain the schema information of the table within a period of time. The detailed process is as follows:
```

Comment:
```suggestion
When a new consumer joins the consumer group, it might start consuming from the middle, so it might miss earlier DDL and BOOTSTRAP messages of the table. In this case, the consumer might receive some DML messages before obtaining the schema information of the table. Therefore, the consumer needs to wait for a period of time until it receives the DDL or BOOTSTRAP message to obtain the schema information of the table. Because TiCDC sends BOOTSTRAP messages periodically, the consumer can always obtain the schema information of the table within a period of time. The detailed process is as follows:
```